docs/spec/distribution/02_state.md

```diff
@@ -51,15 +52,15 @@ type ValidatorDistInfo struct {
 }
 ```
 
-### Delegation Distribution 
+## Delegation Distribution
 
 Each delegation distribution only needs to record the height at which it last
 withdrew fees. Because a delegation must withdraw fees each time it's
 properties change (aka bonded tokens etc.) its properties will remain constant
 and the delegator's _accumulation_ factor can be calculated passively knowing
-only the height of the last withdrawal and its current properties. 
- 
- - DelegationDistInfo: ` 0x02 | DelegatorAddr | ValOperatorAddr -> amino(delegatorDist)`
+only the height of the last withdrawal and its current properties.
```

Comment:
I think the spec needs to be updated...these don't seem to match. Mind creating an issue? I can tackle it.